cpp/src/gandiva/llvm_generator_test.cc

```diff
@@ -40,22 +41,18 @@ class TestLLVMGenerator : public ::testing::Test {
 // Verify that a valid pc function exists for every function in the registry.
 TEST_F(TestLLVMGenerator, VerifyPCFunctions) {
   std::unique_ptr<LLVMGenerator> generator;
-  auto status = LLVMGenerator::Make(TestConfiguration(), &generator);
-  EXPECT_TRUE(status.ok()) << status.message();
+  ASSERT_OK(LLVMGenerator::Make(TestConfiguration(), &generator));
 
   llvm::Module* module = generator->module();
   for (auto& iter : registry_) {
-    llvm::Function* fn = module->getFunction(iter.pc_name());
-    EXPECT_NE(fn, nullptr) << "function " << iter.pc_name()
-                           << " missing in precompiled module\n";
+    EXPECT_NE(module->getFunction(iter.pc_name()), nullptr);
```

Comment:
having the name in stderr is helpful for debugging.

Reply:
I recommend to set this in your `.gdbinit`

```
set environment GTEST_BREAK_ON_FAILURE=1
```

Then you run the failing test under gdb, it'll break at the first failing test.

Reply:
He might mean that the problem may be more evident from looking at test log files :) I think it's OK for now, if it becomes an issue we can improve

Reply:
yes, wes - I meant easier to get error from log files or travis output !. but, we'll only hit this when adding new functions - so, it's obvious anyway.